common/src/main/java/com/wynntils/wc/utils/lootrun/objects/LootrunUncompiled.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wc.utils.lootrun.objects;
+
+import it.unimi.dsi.fastutil.Pair;
+import java.io.File;
+import java.util.List;
+import java.util.Set;
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.phys.Vec3;
+
+public record LootrunUncompiled(List<Vec3> points, Set<BlockPos> chests, List<Pair<Vec3, Component>> notes, File file) {
```

Comment:
since this and `LootrunInstance` are so small and only used within `LootrunUtils`, it might be better to declare them as inner classes instead of dedicating 2 files for their collective ~5 lines of substance

Reply:
ah, hadn't taken a look at the command class before writing this - but since i think the command class _shouldn't_ be handling these in the first place, it still applies

Reply:
Moved almost all classes, except `LootrunUncompiled`. That class is used in commands as well and there is not a clean way to not really use them.

Reply:
left some comments in the command class on how i think it could be done. my philosophy behind a util class is that any data handling should be done entirely within that class - having to extract data and modify it from outside the class isn't great coding practice and ideally shouldn't even be possible

Reply:
"data encapsulation". Yes, it's a generally good coding style to get the right level of coupling. (I have not looked at the latest changes wrt this in this PR yet.)

But. To me a "util" class is more of a "toolbox" kind of thing. The prototypical Util class contains a number of vaguely related, static, utility methods, with no local data. Like `java.util.Objects`. 

That's actually one thing with how the design has shifted in Artemis that I'm not at all fond of -- it seems that too much domain knowledge has been put under the `util` package. :-( That was never my intention in the original design sketch.